src/app/query/bitcoin/balance/btc-balance.hooks.ts

```diff
@@ -7,10 +7,14 @@ import { isUndefined } from '@shared/utils';
 
 import { sumNumbers } from '@app/common/math/helpers';
 
-import { useAllSpendableUtxosByAddress } from '../address/utxos-by-address.hooks';
+import { useNativeSegwitUtxosByAddress } from '../address/utxos-by-address.hooks';
 
 export function useGetBitcoinBalanceByAddress(address: string) {
-  const { data: utxos } = useAllSpendableUtxosByAddress(address);
+  const { data: utxos } = useNativeSegwitUtxosByAddress({
+    address,
+    filterInscriptionUtxos: true,
+    filterPendingTxsUtxos: true,
```

Comment:
Why not make default here too?

Reply:
just feel like better to keep it more explicit there